pom.xml

```diff
@@ -186,6 +192,11 @@
       <artifactId>commons-codec</artifactId>
       <version>1.14</version>
     </dependency>
+    <dependency>
```

Comment:
Added json-smart later library version in because we use this independently of nimbus.